packages/amplify-category-auth/src/__tests__/provider-utils/awscloudformation/utils/message-printer.test.ts

```diff
@@ -1,5 +1,6 @@
 import { printSMSSandboxWarning } from '../../../../provider-utils/awscloudformation/utils/message-printer';
 import { BannerMessage } from 'amplify-cli-core';
+import os from 'os';
```

Comment:
https://github.com/aws-amplify/amplify-cli/pull/7663 should land before this PR.

Reply:
Oh yes,I will revert this change back